app/models/user.rb

```diff
@@ -11,6 +11,8 @@ class User < ApplicationRecord
     if: :address_changed?
 
   after_validation :geocode, :reverse_geocode
+  after_create :create_usersetting
+
   has_one :user_setting, dependent: :destroy
 
```

Comment:
**Reporter**: CHECKSTYLE
**Rule**: com.puppycrawl.tools.checkstyle.Metrics/LineLength
**Severity**: INFO
**File**: app/models/user.rb L17

Line is too long. [82/80]

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>*